### PR TITLE
Add api endpoint

### PR DESCRIPTION
**Pull request by ciraganenicole**
In this pull request I:

- Create a new Rails API app called 'hello-rails-back-end'.
- Make sure that your project has a Postgres database set up.
- Create a table for storing your messages and create 5 different greetings.
- Create an API endpoint that selects a random greeting from your table

### CODE REVIEW

File: app/controllers/concerns/response.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module Response
+  extend ActiveSupport::Concern
+
+  def json_response(object, status = :ok)
+    render json: object, status:
+  end
+end
```
Comment (RolandM99) [inline app/controllers/concerns/response.rb:7]: - [ ] Kindly, as you are using Concerns to make your code reusable and reduce its complexity(make it less complicated) make sure that you implement very well the encapsulation that extends the `ActiveSupport::Concern` . For now when I run your project I have this error below:
image
Kindly, fix this issue.

File: app/controllers/api_controller.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# class ApiController < ApplicationController
+#  include Response
+
+#  def random_greeting
+#    json_response({ message: Greeting.find(Greeting.pluck(:id).sample) })
+#  end
+# end
```
Comment (RolandM99) [inline app/controllers/api_controller.rb]: - [ ] Kindly, remove the commented code from your codebase to make it clean.

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hello @ciraganenicole, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Gitflow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by arslanbisharat — APPROVED** [review]
Hi @ciraganenicole,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**